packages/common-ts/src/base-service/config.ts

```diff
@@ -0,0 +1,24 @@
+import { Validator } from './validators'
+
+/**
+ * Cleans a configuration object.
+ *
+ * @param config Configuration object to clean.
+ * @param validators Mapping of config keys to validators.
+ * @returns Cleaned configuration object.
+ */
+export const cleanConfig = (
```

Comment:
envalid was too restrictive, it only allowed string inputs. Created problems for ChugSplash which needs to create a BaseServiceV2 service programmatically and pass in a Provider object directly. Since it wasn't a string, BSV2 complained. This is much simpler and much more flexible.